addons/website/static/src/scss/website.scss

```diff
@@ -673,6 +673,11 @@ table.table_desc tr td {
     }
 }
 
+.carousel-fade {
```

Comment:
I would put this css somewhere [here](https://github.com/odoo/odoo/blob/16.0/addons/website/static/src/scss/website.scss#L519)

Reply:
@msh-odoo I have tried putting css that place but it won't work as `s_carousel` and `carousel-fade` both classes are siblings. So we have to put this css after `s_carousel`.

Reply:
Can't you add ```&.carousel-fade```?